### DISCUSSION TITLE
Hey I cant see entities when using the API

### DISCUSSION THREAD

**Discussion by jharrison123** [FabricMC/fabric-api#2711] [Player support]
Hey so I am running GDLauncher on a Linux Mint system and whenever I load minecraft Fabric with the fabric api, no other mods, I cant see animals, mobs, or any entity with an AI it looks like. Is there any way to patch this?

**Comment by apple502j** [comment]
Screenshots would be welcome. Also does this happen without Fabric API?

**Reply by jharrison123** [reply]
when I have the Fabric API installed I cant see mobs or animals

**Reply by jharrison123** [reply]
I am also using sodium, litium and phosphor

**Comment by jharrison123** [comment]
I can see arrows and minecarts
